tests/e2e/_configs/templates/persistent.yaml

```
@@ -0,0 +1,62 @@
+#
+# .VolumeHandler example:  "nstor-box2:tank01/fs01/pvol01"
+#
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: nexentastor-csi-driver-cs-nginx-persistent
+provisioner: nexentastor-csi-driver.nexenta.com
+---
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: nexentastor-csi-driver-pv-nginx-persistent
+  labels:
+    name: nexentastor-csi-driver-pv-nginx-persistent
+{{with .VolumeHandler -}}
```

I think .VolumeHandle would make more sense, since that is the name used by k8s
```suggestion
{{with .VolumeHandle -}}
```